README.md

```diff
@@ -20,14 +20,21 @@ Scripts to help work with this repository
 5. tofu_delete_workspace.sh: Delete workspace.
 6. tofu_list_workspace.sh: List available workspaces.
 7. tofu_plan.sh: Generate execution plan to be deployed.
-8. tofu_apply.sh: Deploy the plan to the cloud provider.
-9. tofu_destroy.sh: Delete the deployment from the cloud provider.
-10. tofu_output.sh: Show deployment output.
-11. aws_update_kubeconfig.sh: Update kubectl config.
-12. kubectl_connect_falkordb_master.sh: Port forward into the FalkorDB master node.
-13. kubectl_connect_grafana.sh: Port forward into the grafana gui.
-14. kubectl_connect_prometheus.sh: Port forward into the prometheus gui.
-15. kubectl_connect_alertmanager.sh: Port forward into the alert manager gui.
+8. tofu_plan_aws.sh: Generate execution plan to be deployed for the AWS target.
+9. tofu_plan_k8s.sh: Generate execution plan to be deployed for the K8S target.
+10. tofu_apply.sh: Deploy the plan to the cloud provider.
+11. tofu_apply_aws.sh: Deploy the AWS target to the cloud provider.
+12. tofu_apply_k8s.sh: Deploy the K8S target to the cloud provider.
+13. tofu_destroy.sh: Delete the deployment from the cloud provider.
+14. tofu_output.sh: Show deployment output.
+15. tofu_show.sh: Show the state configuration.
+16. tofu_test.sh: Run Tofu tests.
+17. aws_update_kubeconfig.sh: Update kubectl config.
+18. kubectl_connect_falkordb_master.sh: Port forward into the FalkorDB master node.
+19. kubectl_connect_grafana.sh: Port forward into the grafana gui.
+20. kubectl_connect_prometheus.sh: Port forward into the prometheus gui.
+21. kubectl_connect_alertmanager.sh: Port forward into the alert manager gui.
+22. gcp_update_kubeconfig.sh: Update kubectl config. Args: 1=cluster-name, 2=region, 3=project-name 
```

Comment:
please add description for each forlder/subfolder in the gcp

Reply:
Added